libs/plugins-runtime/src/lib/api/index.ts

```diff
@@ -212,6 +212,11 @@ export function createApi(context: PenpotContext, manifest: Manifest): Penpot {
       return context.selection;
     },
 
+    set selection(value: PenpotShape[]) {
+      checkPermission('content:read');
```

Comment:
I'm not sure if this is a read event. It doesn't change the file, but it's interacting with it. Because I'm not 100% sure if `content:write` is better, I'm going to approve it.

Reply:
I was thinking about this. The selection is really part of the "reading" because you're not modifying the content. Probably if we face this doubt on further apis we'll need to create a new category for "interactions" or something like that.